.travis.yml

```diff
@@ -272,6 +272,7 @@ before_install:
         export LD_LIBRARY_PATH=$CONDA_PREFIX/lib;
         export SYMPY_STRICT_COMPILER_CHECKS=1;
         conda clean --all;
+        pip install clang;
```

Comment:
This should be installed in the optional dependency build only.

Also, it should install with conda if possible. If this is missing from conda-forge, let's work on adding it.